Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java

```diff
@@ -549,6 +549,7 @@ protected void onStop() {
         if (mConnection.isBound()) {
             SharedPreferences.Editor editor = prefs.edit();
             editor.putInt(Preferences.PREF_SOUNDSYSTEM_MASTERVOLUME, ttclient.getSoundOutputVolume());
+            editor.putInt(Preferences.PREF_SOUNDSYSTEM_MICROPHONEGAIN, ttclient.getSoundInputGainLevel());
```

Comment:
If channel is configured with `AudioPreprocessor` (in TeamTalkService line 799) then you'll end up saving the `AudioPreprocessor` gain value. `Preferences.PREF_SOUNDSYSTEM_MICROPHONEGAIN` should only be saved if the user changes the value.